internal/sql/repository/helper/AppListingRepositoryQueryBuilder.go

```diff
@@ -282,9 +282,26 @@ func (impl AppListingRepositoryQueryBuilder) buildAppListingWhereCondition(appLi
 		whereCondition = whereCondition + "and dga.deployment_group_id = " + strconv.Itoa(appListingFilter.DeploymentGroupId) + " "
 	}
 	//add app-status filter here
+	var appStatusWithoutNotDeployed []string
+	var isNotDeployedFilter bool
```

Comment:
change isNotDeployedFilter -> isNotDeployedFilterApplied this is more meaningful